fs/expand_glob.ts

```diff
@@ -253,7 +253,7 @@ export function* expandGlobSync(
   let fixedRoot = isGlobAbsolute
     ? winRoot !== undefined ? winRoot : "/"
     : absRoot;
-  while (segments.length > 0 && !isGlob(segments[0])) {
+  while (segments.length > 0 && !isGlob(segments[0]!)) {
```

Comment:
Note: this non-null assertion is justified because we know that `segments.length > 0`.

Reply:
Also, a non-null assertion is fine here because we already know `segments: string[]`.